client/interfaces.go

```diff
@@ -64,9 +64,18 @@ type Interfaces struct {
 	Slots []Slot `json:"slots"`
 }
 
+// InterfaceInfo holds information about a given interface and its instances.
+type InterfaceInfo struct {
+	MetaData InterfaceMetaData `json:"meta-data"`
+	Plugs    []Plug            `json:"plugs,omitempty"`
+	Slots    []Slot            `json:"slots,omitempty"`
+}
+
 // InterfaceMetaData contains meta-data about a given interface type.
 type InterfaceMetaData struct {
-	Description string `json:"description,omitempty"`
+	Summary          string `json:"summary,omitempty"`
+	Description      string `json:"description,omitempty"`
+	DocumentationURL string `json:"documentation-url,omitempty"`
```

Comment:
DocURL (doc-url)? I'm generally not the one suggesting such shortenings, but that's a very common prefix, 10 less letters, and no reading loss.